### DISCUSSION TITLE
What's ahead for Version 3?

### DISCUSSION THREAD

**Discussion by tacman** [liip/LiipImagineBundle#1545] [General]
What's in the version 3 (development) branch?  I see that it's behind version 2, including the PR that allows Symfony 7.

If you're looking for testers for Version 3, what new features should we be watching out for?

**Comment by dbu** [comment]
thanks for the ping. i merged 2 to 3 in #1546 

the most visible difference is that we start to use the `src/` and `tests/` top level folders. otherwise there is a bunch of things in the [milestone](https://github.com/liip/LiipImagineBundle/issues?q=is%3Aopen+is%3Aissue+milestone%3A3.0.0) but i don't seem to find the time to work on it, and other people neither.
